plugins.txt

```diff
@@ -17,10 +17,7 @@ lockable-resources:2.3
 maven-plugin:3.2
 metrics:4.0.2.2
 monitoring:1.75.0
-openshift-client:1.0.23
-openshift-login:1.0.12
 openshift-pipeline:1.0.55
```

Comment:
This one is actually no longer provided by the upstream image, so we're holding it here as we might be using it in places.